internal/pkg/convert/node.go

```diff
@@ -94,7 +95,29 @@ func (node *NodeLLB) context(root llb.State) llb.State {
 	)
 }
 
+func (node *NodeLLB) convertPlatform(platform string) (v1.Platform, error) {
+	switch platform {
+	case "", "linux/amd64":
```

Comment:
why does an empty one default to amd64? if building on arm64, it should default to arm64 (or simply it should not be set)

Reply:
yeh, makes sense